subdomains/docs/_guide/getting-started.md

```diff
@@ -26,6 +28,23 @@ For Windows, [download and run the Windows installer](https://github.com/volta-c
     <li>Run Volta with elevated privileges (not recommended)</li>
 </ul>" %}
 
-### Windows Subsystem for Linux
+#### Windows Subsystem for Linux
 
 If you are using Volta within the Windows Subsystem for Linux, follow the Unix installation guide above.
+
+
+## Select a default Node version
+
+This is the version that Volta will use everywhere outside of projects that have a pinned version. The `volta pin node` command will use this version by default.
```

Comment:
minor: I don't believe the 2nd sentence is totally accurate. `volta pin node` works just like `volta install node`—you can pass it an explicit version or without one it defaults to the latest LTS release at the time you run the command. Choosing a global doesn't impact subsequent pinning.